.github/workflows/e2e.yml

```
@@ -27,11 +27,34 @@ on:
+  # pair with the temporary ci-downloads release via pnpm ci:download).
+  # `version` pins the resolver so the run works with the vendor API fully
+  # down. A partial dispatch skips record-validation (see that job).
   workflow_dispatch:
+    inputs:
+      browser:
+        description:
+          'Single browser to test (librewolf|floorp|zen|waterfox); empty = default matrix'
+        type: choice
+        default: ''
+        options:
+          - ''
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)